# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1360617794 18000
#      Mon Feb 11 16:23:14 2013 -0500
# Branch stable
# Node ID f47b8b3e2442dadc70240ecdf02cbf7b1c87b9da
# Parent  cc5a7d1233f3acea85648baeb754fc0e8f225225
Added tag rc-3-6-4-2 for changeset cc5a7d1233f3

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -61,8 +61,9 @@ 64d9f33313cc8c691974bcd123357e24bccbabdc
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3.6.0
 95c43fc8dbe1a07a46fefb3372df5b2309d874fd rc-3-6-1-0
 f947d2922febf12dcd1fb6e21b356756ecb54e55 rc-3-6-2-0
 4460c4fb20e6a5d3b1972fa737d4e00eb921545a rc-3-6-2-2
 551566201318bf615b27c60ccf9368f4844008bd release-3-6-2
 a95432e7309ca6fc776c02939264bb6d443f3525 release-3-6-3
 2e8eb9ac43a5f8cfaf0423814a312ed47cb80485 rc-3-6-4-0
 858cbf6fc2ec1c232f5cf1d75dc344439b39a89c rc-3-6-4-1
+cc5a7d1233f3acea85648baeb754fc0e8f225225 rc-3-6-4-2
